work/fifth/index.html

```diff
@@ -99,6 +99,26 @@
       // .change() セレクトボックスの値の変更をキャッチする関数
       // $().each(function(index, element){}) DOM要素を1つずつループ処理をする。
       // eachには引数を二つ持たせることができ、第一引数はDOM要素のインデックス番号、第二引数ではDOM要素自体が取得できます
+      
+      $('select').change(filteringCategory)
+
+      function filteringCategory(){
+        $('.list li').hide()
+        var option = $('option:selected').val()
+        
+        $('.list li').each((index,element) => {
+          showList(index,option,element)
+        })
```

Comment:
[must]
jsのコーディング規約に反してます！
gizumoWIkiの時に指摘されなかったですか？
```suggestion
        $('.list li').each((index, element) => {
          showList(index, option, element)
        })
```

Reply:
今回だけスペースを開けるのを忘れていたので、スペース追加しました。